source/locale/zh_CN/LC_MESSAGES/tutorial/002_configurations.po

```diff
@@ -0,0 +1,105 @@
+msgstr ""
+
+#: ../../source/tutorial/002_configurations.rst:15
+msgid "Advanced Configurations"
+msgstr "高级配置"
+
+#: ../../source/tutorial/002_configurations.rst:18
+msgid "Defining Parameter Spaces"
+msgstr "定义参数空间"
+
+#: ../../source/tutorial/002_configurations.rst:20
+msgid "Optuna supports five kinds of parameters."
+msgstr "Optuna 支持五种类型的参数。"
+
+#: ../../source/tutorial/002_configurations.rst:43
+msgid "Branches and Loops"
+msgstr "分支 (Branches) 与 循环 (Loops)"
+
+#: ../../source/tutorial/002_configurations.rst:45
+msgid "You can use branches or loops depending on the parameter values."
+msgstr "根据不同的参数值，你可以选择使用分支或者循环。"
+
+#: ../../source/tutorial/002_configurations.rst:74
+msgid ""
+"Please also refer to `examples "
+"<https://github.com/optuna/optuna/tree/master/examples>`_."
+msgstr "更多例子见 `examples <https://github.com/optuna/optuna/tree/master/examples>`_."
+
+#: ../../source/tutorial/002_configurations.rst:78
+msgid "Note on the Number of Parameters"
+msgstr "参数个数的注意事项"
+
+#: ../../source/tutorial/002_configurations.rst:80
+msgid ""
+"The difficulty of optimization increases roughly exponentially with "
+"regard to the number of parameters. That is, the number of necessary "
+"trials increases exponentially when you increase the number of "
+"parameters, so it is recommended to not add unimportant parameters."
+msgstr "随着参数个数的增长，优化的难度约呈指数增长。也就是说，当你增加参数的个数的时候，优化所需要的 trial 个数会呈指数增长。因此我们推荐不要增加不必要的参数。"
+
+#: ../../source/tutorial/002_configurations.rst:84
+msgid "Arguments for `Study.optimize`"
+msgstr "`Study.optimize` 的参数"
+
+#: ../../source/tutorial/002_configurations.rst:86
+msgid ""
+"The method :func:`~optuna.study.Study.optimize` (and ``optuna study "
+"optimize`` CLI command as well) has several useful options such as "
+"``timeout``. For details, please refer to the API reference for "
+":func:`~optuna.study.Study.optimize`."
+msgstr ":func:`~optuna.study.Study.optimize` (还有命令 ``optuna study optimize``) 有着数个有用的参数，比如``timeout``. 具体细节见 :func:`~optuna.study.Study.optimize` 的API参考资料。"
+
+#: ../../source/tutorial/002_configurations.rst:90
+msgid ""
+"**FYI**: If you give neither ``n_trials`` nor ``timeout`` options, the "
+"optimization continues until it receives a termination signal such as "
+"Ctrl+C or SIGTERM. This is useful for use cases such as when it is hard "
+"to estimate the computational costs required to optimize your objective "
+"function."
+msgstr "**供參考**: 如果既没有给出 ``n_trials`` 也没有给出 ``timeout`` 参数的话，优化过程将一直持续下去，直到接收到一个诸如 Ctrl+C 或 SIGTERM 的终止信号。"
+"这在难以估算优化目标函数所需的计算成本的情况时是有用的。"
+
+#: ../../source/tutorial/002_configurations.rst:96
+msgid "**Total running time of the script:** ( 0 minutes  0.000 seconds)"
+msgstr ""
+
+#: ../../source/tutorial/002_configurations.rst:111
+msgid ""
+":download:`Download Python source code: 002_configurations.py "
+"<002_configurations.py>`"
+msgstr ""
+
+#: ../../source/tutorial/002_configurations.rst:117
+msgid ""
+":download:`Download Jupyter notebook: 002_configurations.ipynb "
```

Comment:
ditto

Reply:
This part is generated by the gallery plugin, so it is probably a better choice to leave it as it is now.